kv/local_sender.go

```diff
@@ -145,7 +145,7 @@ func (ls *LocalSender) Send(call client.Call) {
 			// MaxTimestamp = Timestamp corresponds to no clock uncertainty.
 			header.Txn.MaxTimestamp = header.Txn.Timestamp
 		}
-		store.ExecuteCmd(call.Args, call.Reply)
+		store.ExecuteCmd(client.Call{Args: call.Args, Reply: call.Reply})
```

Comment:
I could, but this was a simple-minded automated change. I'll do a quick search for these and simplify.